perses/annihilation/new_relative.py

```diff
@@ -42,8 +42,9 @@ class HybridTopologyFactory(object):
     """
 
     _known_forces = {'HarmonicBondForce', 'HarmonicAngleForce', 'PeriodicTorsionForce', 'NonbondedForce', 'MonteCarloBarostat'}
+    _known_softcore_methods = ['default', 'amber', 'classic']
 
-    def __init__(self, topology_proposal, current_positions, new_positions, use_dispersion_correction=False, functions=None):
+    def __init__(self, topology_proposal, current_positions, new_positions, use_dispersion_correction=False, functions=None, softcore_method='default', softcore_alpha=0.5, softcore_beta=12.0*unit.angstrom**2):
```

Comment:
With unit-bearing quantities, I think the preferred idiom is to leave these as `None` and reassign them to the default within the function if they are still `None`. Otherwise we run into the [mutable default arguments issue](http://docs.python-guide.org/en/latest/writing/gotchas/).

Reply:
Ah, good point!